com.djoffe.gnukem.json

```diff
@@ -0,0 +1,65 @@
+{
+  "app-id": "com.djoffe.gnukem",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "davegnukem",
+  "rename-desktop-file": "davegnukem.desktop",
+  "rename-icon-file": "davegnukem.png",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=network",
+    "--persist=.gnukem"
+  ],
+  "modules":
+  [
+    {
+      "name":"dave-gnukem",
+      "buildsystem":"autotools",
+      "no-autogen": true,
+      "no-make-install": true,
+      "post-install":
+      [
+        "install -Dvm755 davegnukem -t ${FLATPAK_DEST}/bin/",
+        "install -dv ${FLATPAK_DEST}/share/",
+        "cp -rfv debian/icons/ ${FLATPAK_DEST}/share/",
+        "install -Dm644 debian/desktop/davegnukem.desktop -t ${FLATPAK_DEST}/share/applications/",
```

Comment:
this is installed by upstream makefile

https://github.com/davidjoffe/dave_gnukem/blob/main/Makefile#L82